.circleci/config.yml

```diff
@@ -7,6 +7,7 @@ jobs:
     working_directory: ~/cli
     environment:
       NYC: "yarn exec nyc -- --nycrc-path node_modules/@oclif/nyc-config/.nycrc"
+      FORCE_COLOR: "0"
```

Comment:
This is really not related to this PR, but was necessary to get circle ci to pass. Happy to create a separate PR for this if desired.